styles/main.css

```diff
@@ -356,3 +338,192 @@ footer {
 #form-btn {
   padding: 1rem;
 }
+
+@media (min-width: 768px) {
+  * {
+    margin: 0 auto;
+  }
+
+  main {
+    background: none;
+  }
+
+  header {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    background-image: url('../assets/Header-llustration-desktop@2x.png');
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  #mobile-nav {
+    display: none;
+  }
+
+  #disktop-nav {
+    margin-top: 2rem;
+    display: block;
+  }
+
+  .container .icons li {
+    padding: 1rem;
+  }
+
+  main .cards .card .work-card-li li {
+    height: auto;
+    border: none;
+    background: none;
+    background: white;
+    justify-content: center;
+    margin: 0;
+    color: #36b37e;
+  }
+
+  #disktop-nav li {
+    margin: 1rem;
+    display: inline;
+  }
+
+  .nav-icons {
+    margin: 3rem 0;
+    padding: 5rem;
+    padding: 0;
+  }
+
+  .container .icons {
+    display: flex;
+    justify-content: center;
+  }
+
+  main .cards {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    gap: 10px;
+    margin: 3rem;
+  }
```

Comment:
- Hey, let's kindly adjust our styles to improve our design. the space in the card is too much and the images are missing the src link. Kindly fix this:

*your design*
![image](https://user-images.githubusercontent.com/59397159/236303444-42d5a59f-f679-4ede-b25b-9c086e675635.png)

*your  design*
![image](https://user-images.githubusercontent.com/59397159/236304307-22db021e-f269-4a56-b7e8-44198cb5cea9.png)


*Figma design*
![image](https://user-images.githubusercontent.com/59397159/236303678-1189fc8d-8ade-4af3-9d18-1da3a333d8e9.png)
*Figma design*
![image](https://user-images.githubusercontent.com/59397159/236304471-cf47499b-3d19-4173-aa94-18680120c767.png)